core/src/utils/input-shims/hacks/hide-caret.ts

```diff
@@ -30,6 +30,6 @@ export const enableHideCaretOnScroll = (
   return () => {
     removeEventListener(scrollEl, 'ionScrollStart', hideCaret);
     removeEventListener(scrollEl, 'ionScrollEnd', showCaret);
-    inputEl.addEventListener('ionBlur', onBlur);
+    inputEl.removeEventListener('ionBlur', onBlur);
```

Comment:
```suggestion
    inputEl.removeEventListener('blur', onBlur);
```

This should also be `blur` not `ionBlur`. The `inputEl` is either an `input` or `textarea` element: https://github.com/ionic-team/ionic-framework/blob/89ad63135df764b4bd382500a4dd57220f85b9e4/core/src/utils/input-shims/input-shims.ts#L39

The `ionBlur` event is dispatched from Ionic components such as  `ion-input`, so listening for `ionBlur` will never do anything.

Reply:
Apologies yes I was testing with 'blur' but for some reason when I created the PR I managed to get 'ionBlur' in there :)